internal/storage/storage_handle_test.go

```diff
@@ -116,10 +116,3 @@ func (t *StorageHandleTest) TestNewStorageHandleWithZeroMaxConnsPerHost() {
 
 	t.invokeAndVerifyStorageHandle(sc)
 }
-
-func (t *StorageHandleTest) TestNewStorageHandleWhenUserAgentIsSet() {
-	sc := getDefaultStorageClientConfig()
-	sc.UserAgent = "gcsfuse/unknown (Go version go1.20-pre3 cl/474093167 +a813be86df)"
```

Comment:
you can set different value of sc.UserAgent (other than the default) and keep this test case?

Reply:
Added, thanks!